xml/System.Collections.Generic/Comparer`1.xml

```diff
@@ -67,7 +67,7 @@
   
  For string comparisons, the <see cref="T:System.StringComparer" /> class is recommended over <c>Comparer&lt;String&gt;</c>.  Properties of the <see cref="T:System.StringComparer" /> class return predefined instances that perform string comparisons with different combinations of culture-sensitivity and case-sensitivity.  The case-sensitivity and culture-sensitivity are consistent among the members of the same <see cref="T:System.StringComparer" /> instance.  
   
- For more information on culture-specific comparisons, see the <see cref="N:System.Globalization" /> namespace and [Globalization and Localization](http://msdn.microsoft.com/library/9a59696b-d89b-45bd-946d-c75da4732d02).</para>
+ For more information on culture-specific comparisons, see the <see cref="N:System.Globalization" /> namespace and [Globalization and Localization](~/docs/standard/globalization-localization/index.md).</para>
```

Comment:
Should the links at this point in the file be markdown, or `<see>`? They seem to render ok on the page, but I'm not sure where else this section is used? I've currently left them as markdown, as the original ones.

https://docs.microsoft.com/en-gb/dotnet/api/system.collections.generic.comparer-1?view=netcore-2.0#notes-to-inheritors

Reply:
It's working fine so let's keep as is.